sphinxcontrib/confluencebuilder/storage/translator.py

```diff
@@ -3,43 +3,40 @@
 # Copyright 2018-2020 by the Sphinx team (sphinx-doc/sphinx#AUTHORS)
 
 from __future__ import annotations
+
+import json
+import posixpath
+import re
+import sys
 from contextlib import suppress
-from docutils import nodes
 from functools import wraps
 from pathlib import Path
+
+from docutils import nodes
 from sphinx import addnodes
 from sphinx.locale import _ as SL
 from sphinx.locale import admonitionlabels
+
 from sphinxcontrib.confluencebuilder.compat import docutils_findall as findall
 from sphinxcontrib.confluencebuilder.exceptions import ConfluenceError
 from sphinxcontrib.confluencebuilder.locale import L
-from sphinxcontrib.confluencebuilder.nodes import confluence_parameters_fetch as PARAMS
-from sphinxcontrib.confluencebuilder.std.confluence import CONFLUENCE_DEFAULT_V2_TABLE_WIDTH
-from sphinxcontrib.confluencebuilder.std.confluence import CONFLUENCE_MAX_WIDTH
-from sphinxcontrib.confluencebuilder.std.confluence import FALLBACK_HIGHLIGHT_STYLE
-from sphinxcontrib.confluencebuilder.std.confluence import FCMMO
-from sphinxcontrib.confluencebuilder.std.confluence import INDENT
-from sphinxcontrib.confluencebuilder.std.confluence import LITERAL2LANG_FBMAP_V1
-from sphinxcontrib.confluencebuilder.std.confluence import LITERAL2LANG_FBMAP_V2
-from sphinxcontrib.confluencebuilder.std.confluence import LITERAL2LANG_MAP_V1
-from sphinxcontrib.confluencebuilder.std.confluence import LITERAL2LANG_MAP_V2
-from sphinxcontrib.confluencebuilder.std.confluence import SUPPORTED_CODE_BLOCK_THEMES
+from sphinxcontrib.confluencebuilder.nodes import \
+    confluence_parameters_fetch as PARAMS
+from sphinxcontrib.confluencebuilder.std.confluence import (
+    CONFLUENCE_MAX_WIDTH,
+    FALLBACK_HIGHLIGHT_STYLE, FCMMO, INDENT, LITERAL2LANG_FBMAP_V1,
+    LITERAL2LANG_FBMAP_V2, LITERAL2LANG_MAP_V1, LITERAL2LANG_MAP_V2,
+    SUPPORTED_CODE_BLOCK_THEMES)
 from sphinxcontrib.confluencebuilder.std.sphinx import DEFAULT_HIGHLIGHT_STYLE
-from sphinxcontrib.confluencebuilder.storage import encode_storage_format
-from sphinxcontrib.confluencebuilder.storage import intern_uri_anchor_value
+from sphinxcontrib.confluencebuilder.storage import (encode_storage_format,
+                                                     intern_uri_anchor_value)
 from sphinxcontrib.confluencebuilder.translator import ConfluenceBaseTranslator
-from sphinxcontrib.confluencebuilder.util import convert_length
-from sphinxcontrib.confluencebuilder.util import extract_length
-from sphinxcontrib.confluencebuilder.util import first
-import json
-import posixpath
-import re
-import sys
+from sphinxcontrib.confluencebuilder.util import (convert_length,
+                                                  extract_length, first)
 
 
 def visit_auto_context_decorator():
-    """
-    node visit decorator
+    """Node visit decorator.
```

Comment:
Please do not include/change the existing documentation style (which is mostly unrelated to the change set).